cmd/commands/namespace.go

```diff
@@ -92,3 +97,37 @@ func NamespaceInit(manifests map[string]*core.Manifest, kc *core.KubectlClient)
 
 	return command
 }
+
+func NamespaceCleanup(kc *core.KubectlClient) *cobra.Command {
+	options := core.NamespaceCleanupOptions{}
+
+	command := &cobra.Command{
+		Use:   "cleanup",
+		Short: "cleans up " + env.Cli.Name + " resources in the namespace",
+		Long: fmt.Sprintf(`cleans up %s resources in the namespace and the namespace itself if "--remove-ns" is set`, env.Cli.Name),
+		Example: fmt.Sprintf(`  %[1]s namespace cleanup my-ns
+  %[1]s namespace cleanup my-ns --remove-ns`, env.Cli.Name),
+		Args: ArgValidationConjunction(
+			cobra.ExactArgs(namespaceCleanupNumberOfArgs),
+			AtPosition(namespaceCleanupNameIndex, ValidName()),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			options.NamespaceName = args[namespaceCleanupNameIndex]
+			if options.NamespaceName == "default" && options.RemoveNamespace {
+				return fmt.Errorf("the default namespace cannot be removed")
```

Comment:
It might be worth adding some words to this error message to make it clear that the resources have not being cleaned up.

Reply:
changed to `"cleanup canceled: the default namespace cannot be removed"`

Reply:
Thanks. That's better.